support-frontend/assets/pages/digital-subscription-checkout/components/datePicker/datePickerHelpers.test.js

```diff
@@ -8,53 +8,73 @@ import {
 } from './helpers';
 import { monthText } from 'pages/paper-subscription-checkout/helpers/subsCardDays';
 
+describe('datePickerHelpers', () => {
```

Comment:
ignore the changes in this file, they are from merging main into this branch, no idea what happened there.  This PR is already merged: https://github.com/guardian/support-frontend/pull/2812/files